torchvision/transforms/v2/_container.py

```diff
@@ -9,6 +9,37 @@
 
 
 class Compose(Transform):
+    """[BETA] Composes several transforms together.
+
+    .. betastatus:: Compose transform
+
+    This transform does not support torchscript.
```

Comment:
Should we advertise this at all given that we want to get rid of it?

Reply:
I think so... But that makes me think we also want to document the JIT thingy in most transforms as well. I suggest we add a "JIT support for v2 transforms" in our docs somewhere, and then automatically link to it in the (edited) warning message of the `.. betastatus::` directive?

Reply:
Fine by me.